testing/common.module.ts

```diff
@@ -20,26 +20,27 @@ export function removeToken(): void {
 
 @NgModule({
     imports: [
-        HttpClientTestingModule,
-        JwtModule.forRoot({
-            config: {
-                tokenGetter: getToken,
-                whitelistedDomains: [],
-            }
-        }),
-        AuthModule.forRoot({
-            aapURL: '',
-            tokenGetter: getToken,
-            tokenUpdater: updateToken,
-            tokenRemover: removeToken
-        }),
-        ToastrModule.forRoot({
-          positionClass: 'toast-bottom-center',
-          closeButton: true,
-          maxOpened: 5,
-          preventDuplicates: true,
-          resetTimeoutOnDuplicate: true
-        })
+      HttpClientTestingModule,
+      JwtModule.forRoot({
+        config: {
+          tokenGetter: getToken,
+          whitelistedDomains: [],
+        }
+      }),
+      AuthModule.forRoot({
+        aapURL: '',
+        tokenGetter: getToken,
+        tokenUpdater: updateToken,
+        tokenRemover: removeToken
+      }),
+      ToastrModule.forRoot({
+        positionClass: 'toast-bottom-center',
```

Comment:
Can we just make it top right? I think it would be better.

Reply:
No problem